src/main/resources/wa-task-completion-ia-asylum.dmn

```diff
@@ -174,6 +174,17 @@
           <text>"Auto"</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_00ego89">
+        <inputEntry id="UnaryTests_0mupz2m">
+          <text>"requestRespondentEvidence","requestCaseBuilding","requestReasonsForAppeal","sendDirection","requestResponseReview","requestCaseEdit","requestResponseReview","sendDirectionWithQuestions","requestCmaRequirements","reviewCmaRequirements","updateDetailsAfterCma","requestResponseAmend","requestResponseReview","reviewHearingRequirements","createCaseSummary","generateHearingBundle","decisionAndReasonsStarted","recordAllocatedJudge","changeDirectionDueDate","decideAnApplication"</text>
```

Comment:
does this comma separated list of events have all the events in relation to the task tasktype? For example requestResponseReview is only in the comma separated list 3 times (there are 4 occurrences in the completion dmn event column). Just checking and making sure we don't need to add the events as in line with the tasktype?

Reply:
well spotted. Need to remove duplicates. 
Yes need to check with Helen whether none of the events are eligible for having no task.